myia/impl_py.py

```diff
@@ -0,0 +1,160 @@
+"""Implementations for the debug VM."""
+
+
+from typing import Dict, Callable
+from copy import copy
+from myia import primops
```

Comment:
PEP8; imports should be grouped standard, third party, local, each with a blank line in between.